docs/data-sources/rediscloud_essentials_database.md

```diff
@@ -59,7 +59,8 @@ data "rediscloud_essentials_database" "example" {
 * `modules` A list of database modules, documented below.
 * `latest_backup_status` - A latest_backup_status object, documented below.
 * `latest_import_status` - A latest_import_status object, documented below.
-* `memory_limit_in_gb` - **Only relevant with Pay-As-You-Go databases.** Maximum memory usage for this specific database.
+* `memory_limit_in_gb` - **Only relevant with Pay-As-You-Go databases.** (Deprecated) Maximum memory usage for this specific database. **Deprecated in favor of `dataset_size_in_gb`**
+* `dataset_size_in_gb` - **Only relevant with Pay-As-You-Go databases.** The maximum amount of data in the dataset for this specific database is in GB.
```

Comment:
Did you do this one? I thought we left out PAYG databases since they're already deprecated